### PR TITLE
fix(core): do not clip center-peek controls to make it appear more quickly

### PR DESCRIPTION
**Pull request by CatsJuice**
fix AF-1084

### CODE REVIEW

**Comment by CatsJuice** [comment]
* **#7572** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7572?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @CatsJuice and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/modules/peek-view/view/modal-container.tsx [hunk 1]
```diff
@@ -214,10 +212,9 @@ export const PeekViewModalContainer = forwardRef<
           opacity: [0, 1],
           duration: 100,
         },
-      })
-        .then(() => animateControls(true))
-        .catch(console.error);
+      }).catch(console.error);
     }, 0);
+    setTimeout(() => animateControls(true), 400 - 230 + 150);
```
Comment (forehalo) [inline packages/frontend/core/src/modules/peek-view/view/modal-container.tsx]: any specific meaning of this timeout number?

Comment (CatsJuice) [reply packages/frontend/core/src/modules/peek-view/view/modal-container.tsx]: To make sure  the end time of the `controls` animation is `150ms` later than the `modal`, regardless of the actual `transition-duration` of the controls.

Comment (pengx17) [reply packages/frontend/core/src/modules/peek-view/view/modal-container.tsx]: Can we listen to animationend event instead?

Comment (CatsJuice) [reply packages/frontend/core/src/modules/peek-view/view/modal-container.tsx]: Yes it was triggered after dialog animation finished before, but we just need a delay here to start animating controls.

**Review by pengx17 — APPROVED** [review]

File: packages/frontend/core/src/modules/peek-view/view/modal-container.tsx (hunk 1 above)
Comment (forehalo) [reply packages/frontend/core/src/modules/peek-view/view/modal-container.tsx]: add a comment incase we forget this magic number

Comment (CatsJuice) [reply packages/frontend/core/src/modules/peek-view/view/modal-container.tsx]: Added

```ts
  /**
   * ### Animation timeline:
   * ```plain
   *                                      150ms
   *                                   ⎮ - - - - ⎮
   * dialog:     +--------400ms--------+
   * controls:               +-------230ms-------+
   *             ⎮ - - - - - ⎮
   *            controls delay =
   *             400 - 230 + 150
   * ```
   */
```

**Review by forehalo — APPROVED** [review]

**Review by EYHN — APPROVED** [review]

**Review by forehalo — APPROVED** [review]

**Pull request merged**